modules/AWS/rds/main.tf

```diff
@@ -22,47 +22,49 @@ module "security_group" {
 }
 
 module "db" {
-  source  = "terraform-aws-modules/rds/aws"
-  version = "~> 3.0"
+  source                      = "terraform-aws-modules/rds/aws"
+  version                     = "~> 5.1"
 
-  identifier = var.rds_instance_identifier
+  identifier                  = var.rds_instance_identifier
 
-  create_db_option_group    = false
-  create_db_parameter_group = false
+  create_db_option_group      = false
+  create_db_parameter_group   = false
 
   # All available versions: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts
-  engine               = "postgres"
-  engine_version       = local.engine_version
-  family               = local.family             # DB parameter group
-  major_engine_version = var.major_engine_version # DB option group
-  instance_class       = var.instance_class
+  engine                      = "postgres"
+  engine_version              = local.engine_version
+  family                      = local.family             # DB parameter group
+  major_engine_version        = var.major_engine_version # DB option group
+  instance_class              = var.instance_class
 
-  allocated_storage = var.allocated_storage
-  iops              = var.iops
+  allocated_storage           = var.allocated_storage
+  iops                        = var.iops
 
-  name     = var.db_name
-  username = local.db_master_username
-  password = local.db_master_password
-  port     = 5432
+  db_name                     = var.db_name
+  username                    = local.db_master_username
+  password                    = local.db_master_password
+  create_random_password      = false
```

Comment:
If we are going to change it to false we need to update comments in config.tfvars.
https://github.com/atlassian-labs/data-center-terraform/blob/ef2d9920f88215a57597d937e6ab5ec82caf1eb5/config.tfvars#L174

Reply:
We will still generate and use password. This attribute is for the module itself to generate one